modules/ModularDiffusers/loaders.py

```diff
@@ -9,68 +9,13 @@
 from utils.torch_utils import DEFAULT_DEVICE, DEVICE_LIST, str_to_dtype
 
 from . import components
+from .modular_utils import get_all_model_types, get_model_type_metadata
 
 
 logger = logging.getLogger("mellon")
 logger.setLevel(logging.DEBUG)
 
 
-@dataclass(frozen=True)
```

Comment:
I moved ModularMellonNodeRegistry from diffusers to mellon and so this registry gets absorbed into that https://github.com/huggingface/diffusers/blob/main/src/diffusers/modular_pipelines/mellon_node_utils.py#L703